tensorflow/compiler/jit/tests/auto_clustering_test.cc

```diff
@@ -57,13 +62,35 @@ TEST_F(AutoClusteringTestImpl, KerasImagenetMainGraphMode) {
   //   tensorflow_models/official/resnet/keras:keras_imagenet_main             \
   //   -- --use_synthetic_data --num_gpus=1 --batch_size=117 --train_steps=600 \
   //   --skip_eval=True --logtostderr --enable_xla
-  TF_ASSERT_OK(RunAutoClusteringTest("keras_imagenet_main_graph_mode"));
+  TF_ASSERT_OK(
+      RunAutoClusteringTestWithPbtxt("keras_imagenet_main_graph_mode"));
+}
+
+TEST_F(AutoClusteringTestImpl, OpenSeq2SeqGNMT) {
+  // Model is from https://github.com/NVIDIA/OpenSeq2Seq.
+  // Generated from
+  //
+  // python run.py \
+  // --config_file=example_configs/text2text/en-de/en-de-gnmt-like-4GPUs.py \
+  // --use_xla_jit
+  TF_ASSERT_OK(
+      RunAutoClusteringTestWithGzippedPbtxt("opens2s_gnmt_mixed_precision"));
+}
+
+#if defined(PLATFORM_GOOGLE)
```

Comment:
Why do you need this `#ifdef`?

Reply:
I do not have benchmark::State in my source tree. I thought only Google had it?

I searched around and it appears under core/platform/google and I do not have this directory.

Reply:
Ok.